applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -453,38 +556,41 @@ int le_audio_pause(void)
 	return 0;
 }
 
-int le_audio_send(uint8_t const *const data, size_t size)
+int iso_stream_send(uint8_t const *const data, size_t size, uint8_t iso_chan_idx)
```

Comment:
This should be a static function

Reply:
It should also be moved to above all the non-static functions defined in le_audio.h